src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
```

Comment:
Consider renaming `array` to something more descriptive, like `stateHistory`, to better convey its purpose.